lib/new_relic/telemetry_sdk/clients/client.rb

```diff
@@ -80,6 +89,10 @@ def add_user_agent_product product, version=nil
         log_error "Encountered error adding user agent product", e
       end
 
+      def api_insert_key
+        TelemetrySdk.config.api_insert_key
+      end
```

Comment:
Should this be in the private section also?

Reply:
yes